File: TypeScript/23_Enum/01_ConsoleColorDemo/ConsoleColorDemo.cs
// ConsoleColor 열거형으로 콘솔의 전경색 및 배경색 바꾸기
using System;

class ConsoleColorDemo
{
    static void Main()
    {
        // 전경색
        Console.ForegroundColor = ConsoleColor.Blue;
        console.log("Blue");
        Console.ResetColor();

        // 배경색
        Console.BackgroundColor = ConsoleColor.Yellow;
        Console.ForegroundColor = ConsoleColor.Red;
        console.log("Red");
        Console.ResetColor();
    }
}

File: TypeScript/23_Enum/01_ConsoleColorDemo/ConsoleColorDemo.js
// ConsoleColor 열거형으로 콘솔의 전경색 및 배경색 표현하기
var ConsoleColorDemo;
(function (ConsoleColorDemo) {
    var ConsoleColor;
    (function (ConsoleColor) {
        ConsoleColor[ConsoleColor["Red"] = 0] = "Red";
        ConsoleColor[ConsoleColor["Green"] = 1] = "Green";
        ConsoleColor[ConsoleColor["Blue"] = 2] = "Blue";
    })(ConsoleColor || (ConsoleColor = {}));
    console.log(ConsoleColor.Red); // 0
    console.log(ConsoleColor.Green); // 1
    console.log(ConsoleColor.Blue); // 2
})(ConsoleColorDemo || (ConsoleColorDemo = {}));
